index.html

```diff
@@ -55,6 +55,12 @@ <h2>Privacy Protections Tests</h2>
 	<li><a href='http://privacy-test-pages.glitch.me/privacy-protections/surrogates/'>Surrogates</a></li>
 </ul>
 
+<h2>Autofill</h2>
+<ul>
+	<li><a href="./autofill/modal.html">Autofill form within modal</a></li>
+	<li><a href="./autofill/form-submission.html">Form submission detection</a></li>
```

Comment:
As far as I understand this page will help to test both detection and autofill, so we can add that

```suggestion
	<li><a href="./autofill/form-submission.html">Form submission detection and autofill</a></li>
```